gravitee-apim-portal-webui-next/src/app/app.routes.ts

```diff
@@ -54,7 +55,7 @@ export const routes: Routes = [
       },
     ],
   },
-
+  { path: 'log-in', component: LogInComponent },
```

Comment:
What would happen if we enter that path after being authenticated, do we handle that currently somehow?

Reply:
Oooo good one! We do not currently handle it. I would imagine that if a user is logged in and goes to log-in page, they should be redirected to the homepage.

Reply:
We could use route guard and `CanActivate` to access the page on certain conditions

Reply:
Done